packages/model-viewer/src/three-components/ARRenderer.ts

```diff
@@ -33,6 +33,7 @@ const INIT_FRAMES = 30;
 // estimation, which will be used once available in WebXR.
 const AR_SHADOW_INTENSITY = 0.3;
 const ROTATION_RATE = 1.5;
+const ROTATION_THRESHOLD = 0.05;
```

Comment:
Thanks for the PR! Can you describe why you think this UX is an improvement? I was following more the Google Maps style where both actions happen together. There is already a 30% snap on scaling to make it sticky near 100%. Also I'd always recommend setting `ar-scale: "fixed"` to disable scaling entirely for products that are meant to be seen at 100%.

Reply:
We do use `ar-scale: "fixed"` whenever we need the product scale precision!
I'm checking a lot on the WebXR code to make our migration and when I read the comment of actions being independent it felt awkward trying to rotate a model without scaling, mostly because it's hard to maintain a constant distance while rotating two fingers. I've also realized I personally use fingers from both hands in opposite directions to rotate, which always make the model shrink then expand. The same movement has always worked nicely on scene-viewer where movements are not interchangeable during a two-finger interaction.
On the other hand, a little bit of rotation while scaling feels acceptable so it's mainly for the reasons above.
Maybe I'm being picky on something that's actually a good feature for most users, since there's already a precise 1 finger rotation, so I don't mind leaving this PR for some extra research😁

Reply:
Thanks for the explanation! I'll leave this for some further discussion for now.

Reply:
> I'll leave this for some further discussion for now.
Awesome, thanks for opening it in the thread!

Reply:
Well, I do agree that slight rotation while scaling is not an issue, but accidentally scaling while trying to rotate is not desired. So if I understood correctly it should be definitely an improvement :) (when rotating beyond threshold scaling is disabled, right ?)

Reply:
> when rotating beyond threshold scaling is disabled, right ?
That's it!

Reply:
For me it is clear now and I support the change :) But maybe for the others :) ?

Reply:
https://demos.r2u.io/model-viewer/index.html

Reply:
Online simplified [demo](https://demos.r2u.io/model-viewer/index.html) for testing